astroid/brain/brain_builtin_inference.py

```diff
@@ -892,6 +892,26 @@ def _build_dict_with_elements(elements):
     return _build_dict_with_elements([])
 
 
+def _looks_like_copy_method(node):
+    func = node.func
+    return isinstance(func, nodes.Attribute) and func.attrname == "copy"
+
+
+def _has_copy_method(node):
+    return any(
+        isinstance(node, klass)
+        for klass in [nodes.Dict, nodes.List, nodes.Set, objects.FrozenSet]
+    )
+
+
+def _infer_copy_method(node, context=None):
```

Comment:
```suggestion
def _infer_copy_method(node: nodes.Call, context: Optional[InferenceContext]=None) -> ...:
```

I think the return value is `Iterator[Union[nodes.Dict, nodes.List, nodes.Set, objects.FrozenSet]]`?